examples/binned_signal_2bkg_systematics.py

```
@@ -0,0 +1,557 @@
+#     "mu": evm_dynamic.mu.value,
+#     "norm1": evm_dynamic.norm1.value,
+#     "norm2": evm_dynamic.norm2.value,
+#     "shape1": evm_dynamic.shape1.value,
+# }
+initial_evm_values = evm.tree.pure(evm_dynamic)
+
+# Wrapper to reconstruct Params structure for evermore loss
+from model import Params  # noqa: E402
+
+
+def evm_nll_wrapper(params_dict, static, hists, observation):
```

Ruff (0.14.0)

Unused function argument: `params_dict`

(ARG001)